### PR TITLE
Integrate IModelLibrary API and respond to queries with real information

### PR DESCRIPTION
**Pull request by acron0**

### CODE REVIEW

File: src/witan/workspace/handler.clj [hunk 1]
```diff
@@ -1,54 +1,57 @@
...
 
-(def app
-  (api
-   {:swagger
-    {:ui "/"
-     :spec "/swagger.json"
-     :data {:info {:title "Witan Workspace Query service"
-                   :description ""}
-            :tags [{:name "api", :description "some apis"}]}}}
+#_(defn ws-handler [request]
```
Comment (elisehuard) [inline src/witan/workspace/handler.clj]: commented on purpose?

File: src/witan/workspace/handler.clj [hunk 2]
```diff
@@ -1,54 +1,57 @@
...
-     :components [peer]
-     :query-params []
-     (p/models peer))
-
-   (GET "/predicates" req
-     :summary "Returns a list of predicates available"
-     :components [peer]
-     :query-params []
-     (p/predicates peer))))
+(defroutes app
```
Comment (elisehuard) [inline src/witan/workspace/handler.clj:35]: ah, I see, bypassing ws for now to test response

File: src/witan/workspace/handler.clj (hunk 1 above)
Comment (acron0) [inline src/witan/workspace/handler.clj]: I'll remove this for now

**Comment by elisehuard** [comment]
:shipit:

**Pull request merged**